### PR TITLE
Fixed `bufio.Scanner: token too long' error

### PR DESCRIPTION
**Pull request by jaytaylor**
Fixed `bufio.Scanner: token too long' error caused by long input lines.

- Swapped out usage of bufio.Scanner for bufio.Reader in lib.LineScanner to
  resolve the issue.

- Includes corresponding unit-test proving this fixes the failing case.

### CODE REVIEW

**Comment by tebeka** [comment]
Thanks! I'll attend to it soon-ish

File: lib/lex_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
...0},{"Start":3581,"End":3596,"Reached":0},{"Start":3605,"End":3623,"Reached":0},{"Start":3632,"End":3686,"Reached":0}]},{"Name":"AppInstance.Stream","File":"/Users/anonymous/go/src/github.standard.com/standard/maestro-aws-tools/pkg/tailer/app_instance.go","Start":3735,"End":5960,"Statements":[{"Start":3840,"End":3915,"Reached":0},{"Start":3917,"End":3950,"Reached":0},{"Start":3952,"End":3988,"Reached":0},{"Start":3970,"End":3985,"Reached":0},{"Start":3990,"End":5902,"Reached":0},{"Start":4021,"End":5899,"Reached":0},{"Start":4053,"End":4080,"Reached":0},{"Start":4085,"End":4313,"Reached":0},{"Start":4318,"End":4594,"Reached":0},{"Start":4374,"End":4399,"Reached":0},{"Start":4404,"End":4428,"Reached":0},{"Start":4475,"End":4515,"Reached":0},{"Start":4520,"End":4562,"Reached":0},{"Start":4541,"End":4556,"Reached":0},{"Start":4567,"End":4589,"Reached":0},{"Start":4599,"End":5872,"Reached":0},{"Start":5877,"End":5895,"Reached":0},{"Start":5904,"End":5958,"Reached":0}]},{"Name":"@159:7","File":"/Users/anonymous/go/src/github.standard.com/standard/maestro-aws-tools/pkg/tailer/app_instance.go","Start":4602,"End":5870,"Statements":[{"Start":4615,"End":4635,"Reached":0},{"Start":4641,"End":4
```
Comment (tebeka) [inline lib/lex_test.go]: `test.log` is not commited.

**Comment by tebeka** [comment]
@jaytaylor Thanks again. Can you add `test.log` to git (missed that in my review)

**Comment by jaytaylor** [comment]
Hey @tebeka, I'm not sure I follow;  The constant `testLog` is the content from the test.log file.

**Comment by jaytaylor** [comment]
@tebeka I see - good catch!  I will open another PR to include the file and clean that bit up tomorrow (or as soon as I'm able).

Best,
Jay

**Comment by tebeka** [comment]
The file is missing from git

**Pull request merged**